### PR TITLE
Tickets/dm 40268

### PR DESCRIPTION
**Pull request by beckynevin**
Deployment is running on one pod; we'd like to merge these changes so that we can make a new ticket for doing some testing on the code itself

### CODE REVIEW

File: tests/test_move_embargo_args.py [hunk 1]
```diff
@@ -97,11 +98,105 @@ def setUp(self):
...
+            ids_moved,
+            self.temp_from_path,
+            self.temp_to_path,
+            move=move,
+            log=self.log,
+        )
+
+    def test_after_now_05(self):
+        """
+        Verify that
```
Comment (ktlim) [inline tests/test_move_embargo_args.py:171]: Complete this docstring?

File: src/move_embargo_args.py [hunk 2]
```diff
@@ -82,8 +83,6 @@ def parse_args():
     # If move is true, then you'll need write
     # permissions from the fromrepo (embargo)
     butler = Butler(namespace.fromrepo, writeable=namespace.move)
-    print("temp from path:", namespace.fromrepo)
-    print("temp to path:", namespace.torepo)
     registry = butler.registry
     dest = Butler(namespace.torepo, writeable=True)
     scratch_registry = dest.registry
```
Comment (ktlim) [inline src/move_embargo_args.py:88]: `scratch` may not be a great name, as it's actually the permanent archive :)

File: src/move_embargo_args.py [hunk 3]
```diff
@@ -137,5 +149,13 @@ def parse_args():
         register_dataset_types=True,
         transfer_dimensions=True,
     )
+    if namespace.log == "True":
+        ids_moved = [
+            dt.dataId.full["exposure"]
+            for dt in scratch_registry.queryDatasets(
```
Comment (ktlim) [inline src/move_embargo_args.py:155]: This is going to be a lot more than you expect when it's the production destination.  I think you need to look at `out` to see what was moved.  And then, below, prune only the datasets that were successfully moved in `out`, not all the ones requested.

**Review by ktlim — APPROVED** [review]

File: src/move_embargo_args.py (hunk 3 above)
Comment (beckynevin) [reply src/move_embargo_args.py:155]: Deleted the log of things we want to move, also now pruning only things that were moved to prompt if move == "True"

File: src/move_embargo_args.py (hunk 2 above)
Comment (beckynevin) [reply src/move_embargo_args.py]: Renaming scratch --> prompt

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -3,10 +3,15 @@ on:
   push:
     branches:
     - main
+    - tickets/DM-40268
```
[the remark this answers is not in the document]
Comment (beckynevin) [reply .github/workflows/test.yml:6]: Right, I had this here so we could deploy before it was a PR, thanks for the reminder.

**Pull request merged**